src/server.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
```

Comment:
I just wonder can we try to use modules here?
If I'm not mistaken NodeJS supports ECMAScript modules natively starting from v13.
It's not mandatory but I encourage you to research this topic and probably try to implement it in your app.